src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java

```diff
@@ -210,7 +211,7 @@ public void enterBed(PlayerBedEnterEvent e) {
 		if (!config.isEnabled() || !config.areDaytimeBedsEnabled()) {
 			return;
 		}
-		if (e.getBedEnterResult() == BedEnterResult.NOT_POSSIBLE_NOW || e.getBedEnterResult() == BedEnterResult.NOT_SAFE) {
+		if (BedEnterResult.NOT_POSSIBLE_NOW.equals(e.getBedEnterResult()) || BedEnterResult.NOT_SAFE.equals(e.getBedEnterResult())) {
```

Comment:
I noticed you did this in many other places as well and actually changed it the other way around in some of my commits. Why use equals() on Enums? == is faster and easier to read.